resources/views/entities/components/profile/organisations.blade.php

```diff
@@ -5,7 +5,21 @@
 @endif
 
 <x-sidebar.profile>
-    @include('entities.components.profile._location')
+    @if ($campaign->enabled('locations') && !$model->locations->isEmpty())
```

Comment:
I feel like this was copy-pasted from somewhere else. If that's the case, unify it into a single `_locations` blade field for races, orgs and creatures :-)